packages/component/src/Localization/ar-SA.json

```
@@ -0,0 +1,3 @@
+{
+  "_.comment": "[CI-LOCKED] The content of this file is locked. Contributions are welcomed but it could be delayed as we conduct additional checks."
```

```suggestion
  "_.comment": "[CI-LOCKED] The content of this file is locked. Contributions are welcome, but change validation may be delayed"
```

Please fix for all files